SKY-4308 Add versions endpoint (#163)
amplitude/amplitude-dev-center: docs/experiment/apis/management-api/versions.md

---
title: Management API Versions Endpoints
---

> ------

Reviewer: ```suggestion
```

---
title: Management API Versions Endpoints
---

> | <div class="big-column">Name</div> | Description |

Reviewer: Recommend removing this table, if there is only one row / section.

| --- | --- |
| [List versions](#list-versions) | List versions of all experiments or flags can the management api key has access to. |

------

## List versions

```bash
GET https://experiment.amplitude.com/api/1/versions
```

Fetch a list of versions of all experiments or flags can the management api key has access to, across multiple projects if the key is scoped to them.

### Query parameters

|Name|Description|
|---|----|
|`start`| The ISO 8601 formatted start time of versions to be returned (inclusive).|
|`end`| The ISO 8601 formatted end time to versions to be returned (exclusive).|
|`limit`| The max number of deployments to be returned. Capped at 1000.|
|`cursor`| The offset to start the "page" of results from.|

### Response

A successful request returns a `200 OK` response and a list of versions encoded as JSON in the response body, along with the cursor to next page.

> Versions will be ordered by version creation time descending.

Reviewer: ```suggestion
Versions are ordered by version creation time descending.
```
